pac4j-core/src/main/java/org/pac4j/core/client/IndirectClient.java

```diff
@@ -40,7 +40,7 @@ protected void internalInit(final WebContext context) {
     }
 
     @Override
-    public final HttpAction redirect(final WebContext context) throws HttpAction {
+    public HttpAction redirect(final WebContext context) throws HttpAction {
```

Comment:
I think I could but I need to reconstruct the location URL.
It's really easier to me to proxify two pac4j bean with my scope tenant and I construct the Clients like this: 
`    @Bean
    @TenantScope
    public Clients clients(Google2Client google2Client) {
        String callbackUrl = ServletUriComponentsBuilder.fromCurrentContextPath().path("/callback?client_name=Google2Client").build().toString();
        return new Clients(callbackUrl, google2Client);
    }`